.vscode/wp-parsely.code-workspace

```diff
@@ -39,6 +39,13 @@
 		"[markdown]": {
 			"files.trimTrailingWhitespace": false
 		},
+		// Error Lens
+		"errorLens.exclude": [
+			// Exclude type coverage errors to reduce noise. The errors are
+			// still visible in all other parts of VSCode and will cause
+			// pre-commit hooks to fail.
+			" actually have it. Add more (return|param) types to get over "
```

Comment:
By noise you mean a prominent messages shows by `Error Lens` or something else? (if we are talking about that prominent message then I think we should keep it as this noise is beneficial and forces us to add more type coverage.

Reply:
While writing code, Error Lens shows messages like "Out of X possible param types, only X % actually have it. Add more param types to get over X %".

The problem with this is that:
- Those errors are shown all over the place next to method signatures that cannot be fixed, or would need refactorings of questionable real value.
- Personally for me, they inhibit my eyes when I'm writing code and they get in the way because they move.
- If someone runs the `static-analysis` command, PHPStan will pass. So actually those red warnings represent an error that doesn't really exist when we run `composer static-analysis` for our whole project.
 
Due to all the above, IMO there is no sense in keeping those errors in the editor, since in-editor errors should be useful and those aren't (they are even misleading). Here's a sample screenshot:

![image](https://github.com/Parsely/wp-parsely/assets/23142906/5ed56b3e-f35f-4cf6-9b05-abcb18d83957)

The reason I put it this exclusion the global workspace is that when I put it in my personal settings it doesn't work. Error Lens gives some commands to disable it temporarily so I could use that instead, but then all errors are gone and this is worse.

Reply:
Noting that even when excluded, the function has a red squiggle that shows the error when hovered.